contracts/package.json

```diff
@@ -33,7 +33,7 @@
     "ethereum-waffle": "^3.2.0",
     "ethers": "^5.0.17",
     "fs-extra": "^9.0.1",
-    "hardhat": "^2.6.0",
+    "hardhat": "^2.6.8",
```

Comment:
This warning occurs when using Rinkeby, and the address in question is only registered as `umbra-demo.eth` on Rinkeby's ENS deployment. This implies to me this part of the code is still resolving ENS based on the Rinkeby deployment.
 
<img width="555" alt="Screen Shot 2021-12-08 at 4 06 44 PM" src="https://user-images.githubusercontent.com/1479802/145287378-06fa4639-d3ba-47bf-b8f2-455a5c299b5e.png">

Reply:
1. Oddly I wasn't able to reproduce this, and it told me `umbra-demo.eth` was an invalid address when on Rinkeby. However I think I see the cause, so I pushed ef043e4 which should fix it. Let me know if that works

2. This made realize that for CNS, we still use their rinkeby registry if we're on Rinkeby. Do we want to keep that, or switch it to mainnet for consistency? My first reaction is to stick with mainnet for CNS too, for consistency

Reply:
@mds1

1. Your fix seems to have worked!
2. I agree, always resolving CNS on mainnet is probably best for consistency. Is it a quick change you can tack on to this PR, or should we create a separate issue?

Reply:
Done in 1233ddc